### ISSUE TITLE
The Docker file doesn't build

### ISSUE THREAD

**Issue by carlosjgp** [opentracing-contrib/nginx-opentracing#22]
How to replicate
```
git clone https://github.com/opentracing-contrib/nginx-opentracing.git
cd nginx-opentracing/docker
docker build -t nginx:opentracing .
```
the error displayed is
```
Scanning dependencies of target jaegertracing
[  0%] Building CXX object CMakeFiles/jaegertracing.dir/src/jaegertracing/Config.cpp.o
In file included from /tmp/tmp.FPf1d0pTos/jaeger-cpp-client/src/jaegertracing/Config.h:21:0,
                 from /tmp/tmp.FPf1d0pTos/jaeger-cpp-client/src/jaegertracing/Config.cpp:17:
/tmp/tmp.FPf1d0pTos/jaeger-cpp-client/src/jaegertracing/baggage/RestrictionsConfig.h:24:27: fatal error: yaml-cpp/yaml.h: No such file or directory
 #include <yaml-cpp/yaml.h>
                           ^
compilation terminated.
CMakeFiles/jaegertracing.dir/build.make:62: recipe for target 'CMakeFiles/jaegertracing.dir/src/jaegertracing/Config.cpp.o' failed
make[2]: *** [CMakeFiles/jaegertracing.dir/src/jaegertracing/Config.cpp.o] Error 1
make[1]: *** [CMakeFiles/jaegertracing.dir/all] Error 2
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/jaegertracing.dir/all' failed
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2

```

**Comment by carlosjgp** [comment]
https://hub.docker.com/r/rnburn/nginx-opentracing/tags/
I just saw that there is a new image pushed `v0.2.1` 6 days ago... which docker file did you use to do it?

**Comment by rnburn** [comment]
Thanks it's caused by https://github.com/jaegertracing/cpp-client/issues/36

You can get it to build if you change 
```
&& git clone https://github.com/jaegertracing/cpp-client.git jaeger-cpp-client \
```
to 
```
&& git clone -b v0.1.0 https://github.com/jaegertracing/cpp-client.git jaeger-cpp-client \
```

**Comment by carlosjgp** [comment]
That worked!

Should I create a PR to update the Dockerfile and the documentation?

**Comment by rnburn** [comment]
Yes, please. It probably would be a good idea to change the Dockerfile to pull from the last release of the repositories instead of master.

**Comment by rnburn** [comment]
Fixed in #24

**Comment by mikhailov** [comment]
```
Pulling nginx (opentracing-contrib/nginx-opentracing:latest)...
ERROR: pull access denied for opentracing-contrib/nginx-opentracing, repository does not exist or may require 'docker login'
```

I wonder if this image hasn't been pushed to the docker hub? What's the recommended way to fetch the image without pulling this repo?
